backend/migrate.ts

```
@@ -0,0 +1,486 @@
+    
+    for (const value of feedItems) {
+        try {
+            const item = JSON.parse(value);
+            // Assuming post IDs start with 'post-' or are UUIDs not containing '+'
+            // Adjust this logic based on your actual post ID format if needed
+            if (item.id && (item.id.startsWith('post-') || /^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/i.test(item.id)) && !item.id.includes('+')) {
+                postIds.add(item.id);
+            }
+        } catch (err) {
+            // Skip invalid JSON
+            continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)